static/date/date/css/homepage.css

```diff
@@ -493,17 +493,14 @@ div.calendar-container a {
 }
 
 .vanilla-calendar-day__btn_today:hover {
-    color: var(--textColorLight) !important;
```

Comment:
Why are we changing date css? Does this need to be rebased?

Reply:
I originally committed these changes as separate commits for PR #602 but the PR had already been merged with develop.

https://github.com/Datateknologerna-vid-Abo-Akademi/date-website/commit/f969849fd9d78d07102fcee3d117d096f63484b3

https://github.com/Datateknologerna-vid-Abo-Akademi/date-website/commit/69d14cd156cba4e7f43ef08cd73968415752e8e4